text/0055-remove-unspecified.md

```diff
@@ -0,0 +1,225 @@
+# Explicit Unspecified Entities
+
+## Related issues and PRs
+
+- Related RFCs: [RFC35](https://github.com/cedar-policy/rfcs/pull/35), [RFC47](https://github.com/cedar-policy/rfcs/pull/47)
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2024-02-27
+
+## Summary
+
+Cedar currently supports _unspecified entities_, which are entities of a special, unique type that have no attributes, and are not ancestors or descendants of any other entity in the store. Unspecified entities are intended to act as placeholders for entities that don’t influence authorization (see examples below).
+
+The current implementation of unspecified entities results in several corner cases in our code (especially the validator), which have been difficult to maintain (see discussion on [this recent PR](https://github.com/cedar-policy/cedar/pull/603)). The concept of unspecified entities is (in our opinion) confusing for users, and will only become more confusing once we stabilize the partial evaluation experimental feature, which allows “unknown” entities that are subtly different from “unspecified” entities.
+
+This RFC proposes to change the terminology surrounding unspecified entities, instead calling them "default" entities.
+It also proposes to give an explicit name to the unspecified entity type (`__cedar::Default`) and to require this name to be used in requests and schemas that use this type.
```

Comment:
We had previously discussed `None` and `Null`, but I think "default" better represents how the entity should be used: you should use it when you want to make an authorization request, but the principal/resource don't matter, so you just want _some default value_.

Reply:
Agree `Default` make sense.

Reply:
I like `Default`

Reply:
`Default` also ties in nicely with rust naming conventions

Reply:
`Any` sounds better for me.